drivers/SmartThings/philips-hue/profiles/hue-smart-button.yml

```diff
@@ -0,0 +1,12 @@
+name: HueSmartButton
+components:
+  - id: main
+    capabilities:
+      - id: button
+        version: 1
+      - id: battery
+        version: 1
+      - id: refresh
+        version: 1
+    categories:
+    - name: RemoteController
```

Comment:
Let's use `Button`

Reply:
For all buttons or just the single button/smart button?

Reply:
Adjusted for single button on https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1359, lmk if it should be done for the multi-buttons as well.

Reply:
Looks good. We've been sticking with RemoteControl for more than one button